app/scripts/views/mixins/signin-mixin.js

```diff
@@ -147,19 +148,29 @@ define(function (require, exports, module) {
 
     _engageSignInForm (event) {
       if (event && event.target) {
```

Comment:
Not for this PR, but it almost seems like this could be made into two functions, one for the click handler so you don't have to do the case statement, one for typing into the input fields.